src/lib/__snapshots__/scrape.test.ts.snap

```diff
@@ -0,0 +1,825 @@
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "which makes sense. It's the easy way out of religious commitment. He did once give his philosophy of mankind–which is cryptic and almost meaningless:",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Quotes",
+      "sourceUrl": "http://velvet_peach.tripod.com/apacinoquotes.html",
+      "text": "My philosophy? Man is a little bit better than his reputation, and a little bit worse.",
+      "type": "quote",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "He's just the kind of guy who likes to remain a mystery.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "Pacino politics",
+      "type": "heading",
+    },
+    Object {
+      "sourceTitle": "Al Pacino's Federal Campaign Contribution Report",
+      "sourceUrl": "http://www.newsmeat.com/celebrity_political_donations/Al_Pacino.php",
+      "text": "Al is only slightly less secretive about his political views. But again, it's probably something he doesn't allow interviewers to question him about. He supports Democrats (at least one) financially. Pacino has only given money to one political candidate–Democrat Ralph Waite–but he's contributed to Waite's campaign four different times, to the tune of $4,000.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "President Obama to Award National Medal of Arts to Al Pacino",
+      "sourceUrl": "http://www.hollywoodreporter.com/news/al-pacino-president-obama-national-medal-of-arts-289385",
+      "text": "Democrats support Pacino as well. He was given the National Medal of Arts by President Barack Obama in 2011.",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "Pacino indicates his liberal view of sexual orientation, indicating confusion regarding any sort of intolerance. He said:",
+      "type": "sentence",
+    },
+    Object {
+      "sourceTitle": "Al Pacino – Biography.",
+      "sourceUrl": "http://www.imdb.com/name/nm0000199/bio",
+      "text": "I don't understand the hatred and fear of gays and bisexuals and lesbians…it's a concept I honestly cannot grasp. To me, it's not who you love…a man, a woman, what have you…it's the fact THAT you love. That is all that truly matters.",
+      "type": "quote",
+    },
+    Object {
+      "sourceTitle": undefined,
+      "sourceUrl": undefined,
+      "text": "Maybe that's the part where mankind is a little worse than his reputation.",
+      "type": "sentence",
+    },
+  ],
+  "lastUpdatedOn": "2012-08-30",
+  "name": "Al Pacino",
+  "politicalViews": "Pacino has supported Democrats financially and expressed a liberal view toward sexual orientation.",
```

Comment:
If either the `politicalViews` or `religion` are made of more than one paragraph, will they still get scraped correctly?

Reply:
I've not seen a page with multiple paragraphs for religion and political views. I'll check the scraping results and fix this if necessary.